services/vod-to-live/service/src/domains/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { create } from 'xmlbuilder2';
+import { SECOND_IN_MILLISECONDS } from '../common';
+
+/**
+ * Converts any object to string XML representation.
+ * @param expandedObject - object to convert to XML.
+ * @returns- string containing XML of the object.
+ */
+export const convertObjectToXml = (
+  expandedObject:
+    | {
```

Comment:
I would replace this type with `string | ExpandObject`, unless an import like `import { ExpandObject } from 'xmlbuilder2/lib/interfaces';` breaks anything...